eng/pipelines/templates/jobs/archetype-sdk-client.yml

```
@@ -293,6 +293,9 @@ jobs:
 
   # Run unit tests
   - script: |
+      export TEST_CA_FILE="$(realpath ../ca.pem)"
+      export TEST_CLIENT_CERTIFICATE_PATH="$(realpath ../client.pem)"
+      export TEST_CLIENT_KEY_PATH="$(realpath ../client-key.pem)"
```

Use `env` instead of `export`. Might need to do something with the path like `$(Build.SourcesDirectory)/ca.pem` 
 
```suggestion
  - script: |
      ctest --version
      ctest -VV --output-on-failure
    workingDirectory: build
    displayName: ctest
    env: 
      TEST_CA_FILE: ca.pem
      TEST_CLIENT_CERTIFICATE_PATH: client.pem
      TEST_CLIENT_KEY_PATH: client-key.pem
```